lib/client.js

```diff
@@ -66,7 +66,7 @@ Client.prototype.init = function(clientId, clientKey, options) {
     route: options.route,
     timeout: options.timeout,
     routeClientId: options.route && options.route.client,
-    cache: options.cache || true,
+    cache: typeof options.cache !== 'undefined' ? options.cache : true
```

Comment:
I didn't include a trailing comma since it doesn't match your current lint rules. Should probably be taken care of in a separate PR if you want to start including those.